deku-derive/src/macros/deku_read.rs

```diff
@@ -121,9 +131,19 @@ fn emit_struct(input: &DekuData) -> Result<TokenStream, syn::Error> {
             use ::#crate_::DekuReader as _;
             let mut __deku_cursor = #crate_::no_std_io::Cursor::new(__deku_input.0);
             let mut __deku_reader = &mut deku::reader::Reader::new(&mut __deku_cursor);
-            if __deku_input.1 != 0 {
-                __deku_reader.skip_bits(__deku_input.1)?;
+            let bytes = __deku_input.1 / 8;
```

Comment:
I see this section is being repeated, worth having an `emit_..` function for this new chunk?

Reply:
I was thinking that also, it's weird to codegen this code over and over again instead of something in the `reader`.

Reply:
Updated, I forgot skip_bits was already only used for this.